objects/spread_values.go

```diff
@@ -0,0 +1,42 @@
+package objects
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/d5/tengo/compiler/token"
+)
+
+// SpreadValues represents the result of spreading an object.
+type SpreadValues struct {
+	Value []Object
+}
+
+// TypeName returns the name of the type.
+func (o *SpreadValues) TypeName() string { return "spread-values" }
+
+func (o *SpreadValues) String() string { return fmt.Sprintf("<%s>", o.TypeName()) }
```

Comment:
I think this should actually return an actual string representation something like `[1, 2, 3]...`?

Reply:
I think this should return either the Array representation `[1, 2, 3]` or an Array after being spread `1 2 3` (or perhaps with commas. I don't think it should actually add ellipses at the end of the string representation

Reply:
This type should never make it to user code. It's created and then de-structured within the VM into regular object values. I'd rather keep the `String` implementation as it is here to make that clear. It's the same way we handle `String` for `Iterator` values